playbooks/service/swapfile.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Configure swap files
+  hosts: [ 'debops_service_swapfile', 'debops_swapfile' ]
```

Comment:
@drybjed Are you planning to support both ('debops_service_swapfile', 'debops_swapfile') long term or is the 'debops_swapfile' format tagged for being dropped?